frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -161,6 +133,50 @@ export function UsersStackedBar({ surveyUserStats }: { surveyUserStats: SurveyUs
     )
 }
 
+export function UsersCount({ surveyUserStats }: { surveyUserStats: SurveyUserStats }): JSX.Element {
+    const { seen, dismissed, sent } = surveyUserStats
+    const total = seen + dismissed + sent
+    const labelTotal = total === 1 ? 'Unique user shown' : 'Unique users shown'
+    const labelSent = sent === 1 ? 'Response sent' : 'Responses sent'
+
+    return (
+        <div className="inline-flex mb-4">
+            <div>
+                <div className="text-4xl font-bold">{humanFriendlyNumber(total)}</div>
+                <div className="font-semibold text-secondary">{labelTotal}</div>
+            </div>
+            {sent > 0 && (
+                <div className="ml-10">
+                    <div className="text-4xl font-bold">{humanFriendlyNumber(sent)}</div>
+                    <div className="font-semibold text-secondary">{labelSent}</div>
+                </div>
+            )}
+        </div>
+    )
+}
+
+export function UsersStackedBar({ surveyUserStats }: { surveyUserStats: SurveyUserStats }): JSX.Element {
+    const { seen, dismissed, sent } = surveyUserStats
+
+    const segments: StackedBarSegment[] = [
+        { count: seen, label: 'Unanswered', color: '#1D4AFF' },
+        { count: dismissed, label: 'Dismissed', color: '#E3A506' },
+        { count: sent, label: 'Submitted', color: '#529B08' },
```

Comment:
lets move the labels to const since they are used in more places.
lets also create const for colors with names, so its easier to figure out whats the color when reading the code, eg const red = '#1D4AFF'

Reply:
for the colors, I changed to use tailwind classes instead.

however, for labels, I don't think those are used anywhere else, and they're specific to this component (`UsersStackedBar`) so I don't think we need to add them as consts

Reply:
the labels are repeated on this file at least twice

Reply:
my bad, the refactor eliminates the dupe, so there is no need.